src/AppBundle/Command/UpdateBundlesListCommand.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/**
+ * Update Bundle List Command.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace AppBundle\Command;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UpdateBundlesListCommand extends ContainerAwareCommand
+{
+    /**
+     * @param array $fields
+     * @return string
+     */
+    private function calculateChecksum(array $fields = array()) {
+        $string = '';
+        foreach ($fields as $field) {
+            $string += $field;
+        }
+        return md5($string);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('app:update_bundles_list')
+            ->setDescription('This command updates Bundle List with data gathered from Packagist.org')
+            ->addOption('force', 'f', InputOption::VALUE_NONE, false);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $repository = $this->getContainer()->get('ezpublish.api.repository');
+        $packagistServiceProvider = $this->getContainer()->get('app.packagist_service_provider');
+        $userService = $repository->getUserService();
+        $searchService = $repository->getSearchService();
+        $contentService = $repository->getContentService();
+        $permissionResolver = $repository->getPermissionResolver();
+
+        $user = $userService->loadUserByLogin('admin');
```

Comment:
Sorry, but I don't see any other way. There is nothing like `$repository->sudo` in `$userService`. Obviously, we can create a user (with permissions to create/update content) and use this account instead of `admin`.